frontend/src/layout/FeaturePreviews/FeaturePreviewsModal.tsx

```diff
@@ -119,19 +126,31 @@ function FeaturePreview({ feature }: { feature: EnrichedEarlyAccessFeature }): J
                             }
                         }}
                     />
-                    <LemonButton
-                        type="primary"
-                        onClick={() => {
-                            void submitEarlyAccessFeatureFeedback(feedback).then(() => {
+                    <div className="flex items-center gap-2">
+                        <LemonButton
+                            type="secondary"
+                            onClick={() => {
+                                cancelEarlyAccessFeatureFeedback()
                                 setFeedback('')
-                            })
-                        }}
-                        loading={activeFeedbackFlagKeyLoading}
-                        fullWidth
-                        center
-                    >
-                        Submit feedback
-                    </LemonButton>
+                            }}
+                        >
+                            Cancel
+                        </LemonButton>
+
+                        <LemonButton
+                            type="primary"
+                            onClick={() => {
+                                void submitEarlyAccessFeatureFeedback(feedback).then(() => {
```

Comment:
Any reason not to use an `async` function here?

Reply:
It's not allowed as the onClick doesn't take a promise

Reply:
Since the eslint changes that is